### ISSUE TITLE
Yale Connect WiFi Bridge compatible?

### ISSUE THREAD

**Issue by gazes01logic** [nnance/homebridge-august-locks#19]
**Describe Your Problem:**
Hi, I’m looking to purchase the Yale Connect WiFi bridge as it is currently on a 50% discount and hence get my Conexis L1 lock into HomeKit via homebridge. I was suggested to look at your plug-in as it may be compatible given the similarities between August and Yale. Do you know / think it would work?

Thank you very much!

**Comment by Hulkkih** [comment]
I got it connected to Yale Doorman V2N + Access module + Wifi-bridge. Sadly it didn't work reliably. Got some errors after couple unlocks and plugin crashed.

**Comment by scottrobertson** [comment]
I ended up doing this all via Home Assistant and the August plugin over there. Worked well for me. Just in case anyone else wants to do it.

**Comment by Hulkkih** [comment]
> I ended up doing this all via Home Assistant and the August plugin over there. Worked well for me. Just in case anyone else wants to do it.

You got the lock in Apple Home app? How did you do it exactly?

**Comment by scottrobertson** [comment]
> > I ended up doing this all via Home Assistant and the August plugin over there. Worked well for me. Just in case anyone else wants to do it.
> 
> 
> 
> You got the lock in Apple Home app? How did you do it exactly?

Yeah. Just added it to HA via the August app, and then added a HomeKit  service in HA in Accessory mode.

**Comment by Hulkkih** [comment]
> > > I ended up doing this all via Home Assistant and the August plugin over there. Worked well for me. Just in case anyone else wants to do it.
> > 
> > 
> > You got the lock in Apple Home app? How did you do it exactly?
> 
> Yeah. Just added it to HA via the August app, and then added a HomeKit service in HA in Accessory mode.

Ok, thanks for the info. Have to consider installing HA just for that August plugin.
